### PR TITLE
[KMP] JS CustomMessageType support + build tweaks

### PR DESCRIPTION
**Pull request by wkal-pubnub**
refactor: Internal changes for Chat SDK

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -39,7 +39,7 @@ POM_DEVELOPER_NAME=PubNub
 POM_DEVELOPER_URL=support@pubnub.com
 
 IOS_SIMULATOR_ID=iPhone 15 Pro
-SWIFT_PATH=swift
+#SWIFT_PATH=../swift
```
Comment (marcin-cebo) [inline gradle.properties:42]: Should this be commented?

File: pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/Pubnub.d.kt [hunk 2]
```diff
@@ -420,6 +422,7 @@ open external class PubNub(config: Any /* UUID | UserId */) {
         var message: Any
         var file: `T$5`
         var userMetadata: Any?
+        var customMessageType: String?
```
Comment (marcin-cebo) [inline pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/Pubnub.d.kt:425]: Do we need this? Is there customMessageType related to MessageActionEvent?

File: pubnub-kotlin/pubnub-kotlin-core-api/build.gradle.kts [hunk 3]
```diff
@@ -30,7 +30,7 @@ kotlin {
         if (enableJsTarget) {
             val jsMain by getting {
                 dependencies {
-                    implementation(npm("pubnub", "8.2.8"))
+                    implementation(npm("pubnub", libs.versions.pubnub.js.get()))
```
Comment (marcin-cebo) [inline pubnub-kotlin/pubnub-kotlin-core-api/build.gradle.kts:33]: Nice!

**Review by marcin-cebo — APPROVED** [review]

File: pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/Pubnub.d.kt (hunk 2 above)
Comment (wkal-pubnub) [reply pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/Pubnub.d.kt:425]: this is in FileEvent

**Comment by wkal-pubnub** [comment]
@pubnub-release-bot release kotlin as v10.3.1

**Pull request merged**